website/app/(examples)/previews/dialog-next-router/test-browser.ts

```diff
@@ -1,5 +1,6 @@
 import type { Page } from "@playwright/test";
-import { expect, test } from "@playwright/test";
+import { expect } from "@playwright/test";
+import { test } from "../../../../../examples/test-utils.ts";
```

Comment:
The `examples` package.json has an `exports` field:

```json
"exports": {
  "./*": "./*"
}
```

So I wonder if you could import from `"examples/test-utils.ts"`.

Reply:
Tried, didn't work.